noir/noir-repo/noir_stdlib/src/hash.nr

```diff
@@ -6,11 +6,12 @@ mod pedersen;
 use crate::default::Default;
 use crate::uint128::U128;
 
-#[foreign(sha256)]
 // docs:start:sha256
 pub fn sha256<N>(input: [u8; N]) -> [u8; 32]
 // docs:end:sha256
-{}
+{
+    crate::sha256::digest(input)
+}
 
 #[foreign(sha256)]
```

Comment:
This is still calling the old blackbox function.

Reply:
sha256() is not calling the old one, or do you mean sha256_slice()?

Reply:
Yep, I linked to the line with `#[foreign(sha256)]` on it. It's just showing a few lines above as well.

Reply:
I believe we do not support hash of slices, do we?
I'd rather remove it then.

Reply:
It seems like we do support it based on the other functions below.

Reply:
We should discuss about the removal of hash of slices.